### PR TITLE
trajectories files moved to pam_configuration

### PR DESCRIPTION
**Pull request by vincentberenz**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

This repository used to host the recorded ball trajectories. 
These files have been moved to pam_configuration.
The python code and the CMakeLists.txt have been updated accordingly.

## How I Tested

Imported context, created an instance of BallTrajectories, and checked its size was the expected one.

## I fulfilled the following requirements

This package will be refactored soonish to follow the guidelines

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@
   <depend>pybind11_vendor</depend>
   <depend>shared_memory</depend>
   <depend>mpi_cmake_modules</depend>
+  <depend>pam_configuration</depend>
```
Comment (luator) [inline package.xml:20]: Unless the config files are needed at build-time `<exec_depend>` should be enough.

**Review by luator — APPROVED** [review]

**Comment by simonguist** [comment]
What's the best way to host multiple datasets of ball trajectories with pam_configuration?

**Review by simonguist — APPROVED** [review]

**Comment by vincentberenz** [comment]
> What's the best way to host multiple datasets of ball trajectories with pam_configuration?

pam_configuration does not do anything special, it just hosts the files and copy them to ~/.mpi-is/pam during compilation.

The simplest way to add trajectories are to add them here:

https://github.com/intelligent-soft-robots/pam_configuration/tree/master/config/context/ball_trajectories

If we want to keep separate sets of trajectories, then we have to put them in subfolders and update the code here:

https://github.com/intelligent-soft-robots/context/blob/vberenz/config/python/context/ball_trajectories.py#L7

But the cleanest way would be to keep all trajectories in a hdf5 file.  A hdf5 file is a file that has an internal structures which is "folder like", so sets can be separated. The bonus is that metadata can be added. For now the record period of 10ms is hard coded in python, which is not good. Would be better as metadata.

https://docs.h5py.org/en/stable/

File: package.xml (hunk 1 above)
Comment (vincentberenz) [reply package.xml]: indeed

**Pull request merged**